src/pretix/base/metrics.py

```diff
@@ -0,0 +1,150 @@
+import redis
+
+ns_prefix = "pretix_metrics_"
```

Comment:
As this is a constant, I'd prefer an uppercase naming, ``NS_PREFIX`` or maybe to be more clear ``REDIS_KEY_PREFIX``.